openspp_dms_ftp_server/clients.py

```diff
@@ -35,9 +35,9 @@ def login(self, username: str, password: str) -> bool:
         response = requests.post(
             url=url, json=data, headers={"Content-Type": "application/json"}
         )
-        data = response.json()
-        if not response.status_code == 200:
+        if response.status_code != 200:
             raise OpenSPPClientException(f"Unsuccessful login. Response data: {data}")
```

Comment:
I believe that the `data` in the exception should be `response.json()`. As written now, it will print the request data.

Reply:
Hi @kneckinator , I fixed the variable names to avoid this.